src/common/transformations/src/transformations/resolve_names_collisions.cpp

```diff
@@ -6,12 +6,27 @@
 
 #include <algorithm>
 #include <memory>
-#include <numeric>
 
 #include "itt.hpp"
-#include "openvino/op/parameter.hpp"
-#include "openvino/op/result.hpp"
-#include "openvino/op/sink.hpp"
+#include "openvino/op/util/multi_subgraph_base.hpp"
+
+namespace {
+
+void collect_name_collisions_map(const std::shared_ptr<ov::Model>& model,
+                                 std::unordered_map<std::string, std::list<ov::Node*>>& name_collisions_map) {
+    for (const auto& node : model->get_ordered_ops()) {
+        // Collect a names collision map for all nodes in the graph
+        const auto& friendly_name = node->get_friendly_name();
```

Comment:
can it be a case that `friendly_name` is empty?

Reply:
->get_friendly_name() method will return autogenerated name
m_friendly_name can be empty